iree/vm/context.c

```diff
@@ -37,6 +38,32 @@ struct iree_vm_context {
 
 static void iree_vm_context_destroy(iree_vm_context_t* context);
 
+// Runs a single `() -> ()` function from the module if it exists.
+static iree_status_t iree_vm_context_run_function(
+    iree_vm_stack_t* stack, iree_vm_module_t* module,
+    iree_string_view_t function_name) {
+  IREE_TRACE_ZONE_BEGIN(z0);
+
+  iree_vm_function_call_t call;
+  memset(&call, 0, sizeof(call));
+  if (!iree_status_is_ok(iree_vm_module_lookup_function_by_name(
+          module, IREE_VM_FUNCTION_LINKAGE_EXPORT, function_name,
+          &call.function))) {
+    // Function doesn't exist (or otherwise).
+    IREE_TRACE_ZONE_END(z0);
+    return IREE_STATUS_OK;
```

Comment:
Should this be returning the error from `iree_vm_module_lookup_function_by_name` instead of `IREE_STATUS_OK`? Maybe add a comment for why the error is ignored?